spec/person_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'spec_helper'
+
+describe Person do
+  before :each do
+    @person = Person.new(10, 30, 'Abdulali')
+    @unknown = Person.new(10, 10)
+  end
+
+  context 'When testing a Person class' do
+    it 'takes three parameters and returns a Person object' do
+      expect(@person).to be_an_instance_of Person
+    end
+
+    it 'should return the correct age = 30' do
+      expect(@person.age).to eql 30
+    end
```

Comment:
- [ ] Please write unit tests for all the `add_rental` method in the `Person` class as you are required to test all methods in all classes 👍